### PR TITLE
Carry 19085 -- Improve & cleanup documentation comments 

### PR DESCRIPTION
**Pull request by cpuguy83**
Carry #19085

### CODE REVIEW

**Comment by icecrime** [comment]
LGTM

File: builder/dockerfile/builder.go [hunk 1]
```diff
@@ -173,11 +173,15 @@ func (b *Builder) Cancel() {
 	})
 }
 
-// BuildFromConfig will do build directly from parameter 'changes', which comes
-// from Dockerfile entries, it will:
-// - call parse.Parse() to get AST root from Dockerfile entries
-// - do build by calling builder.dispatch() to call all entries' handling routines
-// TODO: remove?
+// BuildFromConfig will do build directly from parameter 'changes',
```
Comment (MHBauer) [inline builder/dockerfile/builder.go]: I suggest `BuildFromConfig builds directly from changes, treating it as if it were the contents of a Dockerfile.`

File: builder/dockerfile/builder.go [hunk 2]
```diff
@@ -173,11 +173,15 @@ func (b *Builder) Cancel() {
...
+// which is treated like it is coming from a Dockerfile. It will:
+// - Call parse.Parse() to get an AST root for the concatenated Dockerfile entries.
+// - Do build by calling builder.dispatch() to call all entries' handling routines
+//
+// BuildFromConfig is used by the /commit endpoint, with the changes
+// coming from the query parameter of the same name.
+//
+// TODO: Remove?
```
Comment (MHBauer) [inline builder/dockerfile/builder.go]: TODO? Is this saying to remove the function in the future at some point?

Comment (cpuguy83) [inline builder/dockerfile/builder.go]: It's from before this change.

File: builder/dockerfile/builder.go (hunk 1 above)
Comment (MHBauer) [inline builder/dockerfile/builder.go]: I do realize the original is really only adding a sentence below and this is just a formatting change. I don't like that first sentence, so feel free to ignore.

Comment (cpuguy83) [inline builder/dockerfile/builder.go]: updated

**Comment by MHBauer** [comment]
LGTM, punch it.

@thaJeztah do we do docs review on comments?

**Comment by icecrime** [comment]
@MHBauer Nah don't bother :wink:

**Pull request merged**